### ISSUE TITLE
support timeout?

### ISSUE THREAD

**Issue by kzidane** [cs50/style50#67]
`style50` takes too long while checking certain files (e.g., Cloud9's backup files and files with huge amount of data such as that of a student who made an array out of `US.txt` contents and pasted it in one of the source files). 

While some of these files we should certainly ignore by default and others have contents that don't make sense, should we also support something like `--timeout x` that will basically have `style50` spend at most `x` seconds on checking a source file, reporting that the file is too big if the check didn't conclude within this time window? Not sure what that would mean for the overall score though. Any thoughts?

**Comment by cmlsharp** [comment]
This strikes me as being outside the perview of style50. This is easily accomplished by running `timeout x style50 ...`

**Comment by kzidane** [comment]
Right, but that would kill the whole `style50` process, not just skip a file. Would we want that instead?
